packages/lib-user/src/components/shared/MainContent/MainContent.js

```diff
@@ -130,7 +131,9 @@ function MainContent({
           gap='16px'
           justify='end'
         >
-          <button type='button' onClick={() => alert('Coming soon!')}>Generate Volunteer Certificate</button>
+          <Link href={`/users/${source.login}/stats/certificate`}>
```

Comment:
I'll inquire about styling; should it be a (link that looks like a) button per Figma or a link similar to other content section links?

Reply:
Done in 4908653